templates/security/security.phtml

```diff
@@ -57,8 +57,8 @@
                     A security advisory will be released on the Laminas Project
                     site detailing the vulnerability, as well as
                     recommendations for end-users to protect themselves.
-                    Security advisories will be listed at <a href="/security/advisories">https://getlaminas.org/security/advisories</a>,
-                    as well as via a <a href="/security/feed">feed</a> (which
+                    Security advisories will be listed at <a href="<?= $this->url('security.pages', ['action' => 'advisories']) ?>">https://getlaminas.org/security/advisories</a>,
```

Comment:
I think we should use the helper also for the content - the text we have between `<a...>` and `</a>`.

Reply:
If you do, use the `serverurl()` helper to give it a fully qualified URL.